tests/conda_env/test_cli.py

```diff
@@ -83,7 +83,7 @@ def run_env_command(command, prefix, *arguments):
         arguments[1:1] = ['-n', prefix]
     elif command is Commands.ENV_CREATE: # CREATE
         if prefix:
-            arguments[1:1] = ['-f', prefix]
+            arguments[1:1] = ['-n', prefix]
```

Comment:
I think we should test ``prefix`` to see if it is path-like (has slashes, leading periods, is an abspath, etc.).  If it is a path, this should be -p.  If it's a name, it should be -n.

Reply:
I hadn't considered `-p`.  It's also possible I'm not across any overloaded meaning of the terms `prefix` and `name`, which are used interchangeably in the code in places.

None of the existing tests make use of `-p`, so there might be some coverage options there.  Hopefully none of the existing tests pass a path-type `-n` - this arg is positional `prefix`, or so it seems.

Existing tests are mostly creating by name and file, but important to test that the name takes priority in the created env I guess.  Just looked wrong that the prefix was passed to file in this one case, and test callers were bent to fit that.

Eventually these tests need to point out issues in
https://github.com/conda/conda/blob/9a303dcb394fe4e2a76cd97dd379ebc9df7e8679/conda_env/specs/__init__.py#L15
... where remote def and name might be passed in a single kwarg without being able to distinguish between them, causing logic issues in that method.